Sources/StreamChat/Utils/Cached.swift

```diff
@@ -10,8 +10,10 @@ import Foundation
 /// to cache it for certain period of time to prevent unnecessary recomputing of the value.
 @propertyWrapper
 class Cached<T> {
+    typealias ComputeValue = (() -> T)
+    
```

Comment:
Maybe a nit but my opinion is that typealiases like this make it easier to write but harder to read. I'd prefer having it the other way around tbh.

For example, when looking at the following samples, the latter is IMO easier to understand.

```swift
    var projectedValue: ComputeValue { }
    @Atomic private var _cached: T?
```

```swift
    var projectedValue: () -> T { }
    @Atomic private var _cached: T?
```